.github/workflows/image-corelight.yaml

```diff
@@ -0,0 +1,62 @@
+name: Build core/light images
+
```

Comment:
The workflows here are just to start pushing core/light images from master, not yet for releases